utils/search/solr.py

```diff
@@ -426,7 +426,6 @@ def _request(self, query_string="", message=""):
 
         if response.status != 200:
             raise SolrException, response.reason
-        
         return response
 
     def select(self, query_string, raw=False):
```

Comment:
We've been thinking about replacing this file with a maintained 3rd party solr library instead of our custom connection code. Do you think it's a good time replace it as part of this PR, or should we release as-is and then change the library at a later stage?

Reply:
For what I could see this works without much changes, so I think it would be better to do it in two steps.

Reply:
OK, cool. Can you open a new issue for this?